front/app/components/EventCard/DateBlocks/DayAndMonth.tsx

```diff
@@ -6,13 +6,18 @@ import { EventDay, EventMonth } from './styling';
 // hooks
 import useLocale from 'hooks/useLocale';
 
+// utils
+import { showDotAfterDay } from 'utils/dateUtils';
+import { isNilOrError } from 'utils/helperUtils';
+
 interface Props {
   day?: string;
   month?: string | null;
 }
 
 export default ({ day, month }: Props) => {
   const locale = useLocale();
+  const addDotToDay = !isNilOrError(locale) && showDotAfterDay(locale);
```

Comment:
This was an existing bug in the platform which Jelena brought up - I guess for the date blocks on the event card we were missing a " . " after the day. There isn't really a way to get this from the moment library though (just the localized day with the dot for German locales) so it required a custom function.

Reply:
Sure. In any case, given the small scope (only in this component + for de-DE + not the worst problem we have), I think this custom solution is good!